sdk/servicebus/azure-messaging-servicebus/swagger/README.md

```diff
@@ -45,4 +45,94 @@ context-client-method-parameter: true
 customization-class: src/main/java/AdministrationClientCustomization.java
 enable-xml: true
 enable-sync-stack: true
+generic-response-type: true
+custom-strongly-typed-header-deserialization: true
+disable-client-builder: true
+```
+
+### Change Return Types of Subscription REST Methods
+```yaml
```

Comment:
Is it possible to add comments? What are you changing the return type of the subscription methods to? It isn't immediately obvious to me without reading what it used to be and what it will be.